mappings/net/minecraft/entity/EntitySize.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS ahm net/minecraft/entity/EntitySize
+	FIELD a width F
+	FIELD b height F
+	FIELD c constant Z
+	METHOD a grow (F)Lahm;
```

Comment:
This is used just as often to shrink as to grow, and it returns a new object, so maybe `scaled` ?

Reply:
Imo `grow` also implies growth by extending the edges -- `scaled` would seem more accurate